README.md

```diff
@@ -587,6 +587,64 @@ list slicing notation:
 new_table = old_table[0:5]
 ```
 
+## Themes
+
+### Getting Themes
+
+If you want your tables to be even fancier (if that's possible) then you can use color
+themes. All you have to do is import it.
+
+```python
+from prettytable import PrettyTable, THEME
+```
+
+> Note, if you are using Windows, type `pip install prettytable[windows]`, so the color
+> codes get reformatted to your OS. If color does not show for other operating systems,
+> you can still do this and that may fix your problem.
```

Comment:
If I `pip install prettytable[windows]` on non-Windows, will it make a difference?

The `init()` is only done for Windows OS.

https://github.com/tartley/colorama#initialisation says:

> Applications should initialise Colorama using ... `init()`
>
> On Windows, calling init() will filter ANSI escape sequences out of any text sent to `stdout` or `stderr`, and replace them with equivalent Win32 calls.
> 
> On other platforms, calling `init()` has no effect (unless you request other optional functionality; see "Init Keyword Args", below). By design, this permits applications to call init() unconditionally on all platforms, after which ANSI output should just work.
> 
> To stop using Colorama before your program exits, simply call `deinit()`. This will restore `stdout` and `stderr` to their original values, so that Colorama is disabled. To resume using Colorama again, call `reinit()`; it is cheaper than calling `init()` again (but does the same thing).

Some thoughts:

We could just call `init()` for all operating systems.

Or better, put it in a `try`/`except ImportError` for in case colorama isn't installed.

And in fact, we only want to call `init()` when using colour themes. If we're using plain mono, we don't want any extra import overhead.

So maybe only attempt the import/`init()` inside `set_theme`?

We could also do it when `theme != None`, and call `deinit()` when `set_theme(None)`.

(Maybe also guard the import/`init` with a `platform.system() == "Windows"`?)

Reply:
I found a way to implement this finally.
I found this solution in a separate [project](https://github.com/Team-Engage/engine/blob/3de2e93d769252539ab6cabe4f515980e5bc2915/setup.py#L19), and tested it on windows. Should work now, though I must admit the color codes do not translate as smoothly as I was hoping. That should be find though.